VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI, the open source initiative that seeks to improve the current state of Event Driven Architectures (EDA). If you wanna know more about the spec just check the previous video (3) on "What is AsyncAPI". This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project, Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to the Linux Foundation. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, and Microsoft.
+
+## Similarities between AsyncAPI and OpenAPI
+
+AsyncAPI's ultimate goal is to work with EDA architectures as simply as with Rest APIs. For those who don't know what REST APIs are, we will briefly say that interface between systems and that uses HTTP to obtain data or generate operations on that data in all possible formats.
+
+Back to the comparison, the AsyncAPI specification in the asynchronous world is analogous to OpenAPI in the synchronous world. 
+
+Both initiatives share numerous similarities:
+
+- Contract-first philosophy
+- Implementation-agnostic
+- Similar syntax
+- A tooling ecosystem 
+- Extensible
+- They protect information consumption using different security schemes. (Example: user and password, API keys, certificates, OAuth, OIDC, etc.)
+
+## Differences between AsyncAPI and OpenAPI
```

Comment:
biggest difference is 

OpenAPI is for HTTP protocol 
AsyncAPI is for multiple different protocols designed for async communication